### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by aliosgaming**
Some edits to file formatting and wording

### CODE REVIEW

**Comment by paulblessing** [comment]
FWIW these changes (re)introduce some typos as well as undo some work I had done on making wording and capitalization more consistent (using "Mods Manager" which is the title of the app's window rather than "Mod Manager"; using "GitHub" instead of "Github", etc).

**Comment by aliosgaming** [comment]
Yeah my bad I seem to have been working off the previous version at the exact same time this went up. I will try to clean it back up before Zak approves it

**Comment by aliosgaming** [comment]
Okay, most recent commit should have fixed everything? Should be good to go

File: docs/setup/Panacea-ModLoader/index.md [hunk 1]
```diff
@@ -9,124 +9,125 @@
 * [Stability Notes](#you-are-now-ready-to-play-the-kh2-randomizer)
 
 ## Heads up!
-
-* Feel free to go to the [Discord server](https://discord.gg/vKhdwNAmzE) for help with the setup, there is a dedicated #help channel where you can find people to assist you if you're facing any difficulties. (P.S. Please be polite)
-* **You must run the game once** before you start the guide. Otherwise, it may not boot up correctly after the randomizer is installed.
-* **VERIFY GAME FILES** through the Epic Games Launcher for Kingdom Hearts 1.5 + 2.5 Remix (v1.0.0.8_WW), they need to be clean/unpatched files for the extraction process to work.
-* If you have EVER installed the KH2 Randomizer before then please check your `C:\Documents\KINGDOM HEARTS HD 1.5+2.5 ReMIX\scripts\kh2` folder (if you have one) and EMPTY IT.
-* Unofficial copies of the game are not supported, it is strongly recommended you buy and install the game directly from the Epic Games Store.
+* Feel free to go to the [discord server](https://discord.gg/vKhdwNAmzE) for help with the setup, there is a dedicated #help channel where you can find people to assist you if you're facing any difficulties. (P.S. Please be polite)
```
Comment (paulblessing) [inline docs/setup/Panacea-ModLoader/index.md:12]: "discord" -> "Discord"

File: docs/setup/Panacea-ModLoader/index.md [hunk 2]
```diff
@@ -9,124 +9,125 @@
...
+* `You must run the game once` before you start the guide. Otherwise it may not boot up correctly after the randomizer is installed.
+* `VERIFY GAME FILES` through the Epic Games Launcher for Kingdom Hearts 1.5 + 2.5 Remix (v1.0.0.8_WW), they need to be clean/unpatched files for the extraction process to work.
+* If you have EVER installed the KH2 Randomizer before then please check your `C:\Documents\KINGDOM HEARTS HD 1.5+2.5 ReMIX\scripts\kh2 folder` (if you have one) and EMPTY IT.
+* Unofficial copies of the game are not supported, it is strongly recommended you buy and install the game directly from the Epic Game Store.
```
Comment (paulblessing) [inline docs/setup/Panacea-ModLoader/index.md:16]: "Epic Game Store" -> "Epic Games Store"

File: docs/setup/Panacea-ModLoader/index.md [hunk 3]
```diff
@@ -9,124 +9,125 @@
...
 
     ![Target New Seed](../images/Panacea-ModLoader/Target%20New%20Seed.png)
 
-5. Be sure to click on the checkbox next to the seed, then click on **"Build"** and **"Build and Run"** to enable the mod in game. Note the 5 buttons to the right of the mod list. The First button places a mod at the top of the list. The second moves a mod up the list one space, the third moves it down the list by one space. The Green '+' icon is a shortcut to install a new mod, while the Red "-" icon is a shortcut to deleting a mod
-	- **How to install a new seed in the future:** Create a new zip seed file using the generator. Delete the current seed from your Mods Manager. Add the new zip seed to your Mods Manager. Enable the seed and then **"Build and Run"**
+6. Be sure to click on the check box next to the seed, then click on `Build` and `Build and Run` to enable the mod in game. Note the 5 buttons to the right of the mod list. The First button places a mod at the top of the list. The second moves a mod up the list one space, the third moves it down the list by one space. The Green '+' icon is a shortcut to install a new mod, while the Red "-" icon is a shortcut to deleting a mod
```
Comment (paulblessing) [inline docs/setup/Panacea-ModLoader/index.md:105]: "First" -> "first", "Red" -> "red" probably

File: docs/setup/Panacea-ModLoader/index.md [hunk 4]
```diff
@@ -9,124 +9,125 @@
...
 
 __Technical And Stability Notes__
-1. As mentioned earlier in the guide, the Mods Manager/Lua doesn't seem to work if the folder is under a "OneDrive" directory. We don't know exactly why but it's a consistent thing between users. Be sure to move your folders and re-run the setup wizards if you installed in a "OneDrive" directory.
-2. Recommended to set FPS limit to **60**.
-3. Recommended to run game in Windowed/Borderless Fullscreen mode. Fullscreen is stable but the game can crash if you alt-tab out.
-4. Overlays that hook onto the game process (Steam, RivaTuner, NVIDIA Overlay, etc.) and recording programs (OBS, Streamlabs, Xsplit) seem to decrease stability when playing rando. It's recommended to disable them, or in the case of recording programs use window/desktop capture, in order to minimize your chances of a crash.
+1. As mentioned earlier in the guide, the Mods Manager/Lua doesn't seem to work if the folder is under a "OneDrive" directory. We don't know exactly why but it's a conistent thing between users. Be sure to move your folders and re-run the setup wizards if you installed in a "OneDrive" directory.
```
Comment (paulblessing) [inline docs/setup/Panacea-ModLoader/index.md:128]: "conistent" -> "consistent"

File: docs/setup/Panacea-ModLoader/index.md [hunk 5]
```diff
@@ -9,124 +9,125 @@
...
+2. Recommended to set fps limit to `60fps`.
+3. Recommended to run game in windows/borderless windowed mode. Fullscreen is stable but the game can crash if you alt-tab out.
```
Comment (paulblessing) [inline docs/setup/Panacea-ModLoader/index.md:130]: The options are "Windowed" and "Borderless Fullscreen".

image

File: docs/setup/Panacea-ModLoader/index.md [hunk 6]
```diff
@@ -9,124 +9,125 @@
...
+4. Overlays that hook onto the game process (Steam, RivaTuner, Nvidia Overlay, etc) and recording programs (Obs, Streamlabs, Xsplit) seem to decrease stability when playing rando. It's recommended to disable them, or in the case of recording programs use window/desktop capture, in order to minimize your chances of a crash.
```
Comment (paulblessing) [inline docs/setup/Panacea-ModLoader/index.md:131]: "NVIDIA" and "OBS" are their proper names.

File: docs/setup/Panacea-ModLoader/index.md [hunk 7]
```diff
@@ -9,124 +9,125 @@
...
 5. GoA ROM and the Seed Generator will always be compatible with one another. Any other mods beyond that should be compatible as long as they don't overlap on changes. Mods at the top of the list will overwrite mods below them if there's any overlapping files. Feel free to ask in the [Community Discord](https://discord.gg/vKhdwNAmzE) if you aren't sure.
-6. Boss/Enemy Rando is amazing but still a work in progress. Please report any bugs/softlocks/out-of-bounds glitches you encounter to the corresponding Google Form listed in the `#bug-reports` channels of our Discord. 
+6. Boss/Enemy Rando is amazing but still a work in progress. Please report any bugs/softlocks/out-of-bounds glitches you encounter to the coresponding Google Form listed in the #bug-reports channels of our Discord.
```
Comment (paulblessing) [inline docs/setup/Panacea-ModLoader/index.md:133]: "coresponding" -> "corresponding"

**Comment by paulblessing** [comment]
LGTM at this point

**Pull request merged**